operator/build.gradle

```diff
@@ -327,88 +377,160 @@ task prepareArtifacts(type: Exec, dependsOn: [buildContractSamples]) {
+        
+        println 'Waiting for operator deployment to become ready...'
+        // waitForOperatorRunning(namespace, "controller")
+        waitForOperatorRunning(namespace)
+        
+        println 'Waiting for besu statefulSets to become ready...'
+        waitForBesuRunning(namespace, 3, "evm")
+        
+        println 'Waiting for paladin statefulSets to become ready...'
+        waitForPaladinRunning(namespace, 3, "worker")
+        
+        println 'Verifying all scd resources have status "Success"...'
+        waitForAllSuccess(namespace, "scd", 120)
+        
+        println 'Verifying all txn resources have status "Success"...'
+        waitForAllSuccess(namespace, "txn", 60)
+        
+        println 'Verifying all registries are published...'
+        waitForAllRegistriesPublished(namespace, 120)
 
-        verifyResourceCreated(namespace, 'statefulset', 'paladin-node1', "1", 120)
-        verifyResourceCreated(namespace, 'statefulset', 'paladin-node2', "1", 120)
-        verifyResourceCreated(namespace, 'statefulset', 'paladin-node3', "1", 120)
+        printClusterStatus(namespace)
     }
 }
 
 
+// Task to deploy the operator and the Paladin nodes
+task deploy(dependsOn: [installOperatorDefault]) {
+}
 
-// Task to verify all scd resources have status 'Success'
-task verifyAllScdSuccess(dependsOn: verifyPaladin ) {
+// Task to run the e2e tests for the default installation
+// we broke it down to two tasks to allow default installation without tearing down the environment at the end of the test
+task ciOperatorDefault(dependsOn: [deploy]) {
     doLast {
-        println 'Verifying all scd resources have status "Success"...'
-        waitForAllSuccess(namespace, "scd", 120)
+        println 'Uninstalling operator...'
+        uninstallOperator(namespace)
     }
 }
 
-// Task to verify all txn resources have status 'Success'
-task verifyAllTxnSuccess(dependsOn: verifyAllScdSuccess ) {
+task ciOperatorNodeCount(dependsOn: [installOperatorNodeCount]) {
+    def namespace = "ci-node-count"
+    // TODO: Add tests
+    
     doLast {
-        println 'Verifying all txn resources have status "Success"...'
-        waitForAllSuccess(namespace, "txn", 60)
+        println 'Uninstalling operator...'
+        uninstallOperator(namespace)
     }
-} 
+}
 
-// Task to verify all nodes are published
-task verifyAllRegistriesPublished(dependsOn: verifyAllTxnSuccess) {
-    doLast {
-        // println 'Restarting the Operator...'
-        // restartOperator(namespace)
+task ciOperatorPaladinCount(dependsOn: [installOperatorPaladinCount]) {
+    def namespace = "ci-paladin-count"
+    // TODO: Add tests
 
-        println 'Verifying all registries are published...'
-        waitForAllRegistriesPublished(namespace, 120)
+    doLast {
+        println 'Uninstalling operator...'
+        uninstallOperator(namespace)
     }
 }
 
+task ciOperatorRename(dependsOn: [installOperatorRename]) {
```

Comment:
It feels like this is really `ciOperatorCustomName` rather than `ciOperatorRename`? Likewise for `installOperateRename`